### ISSUE TITLE
Add labels to show/hide password buttons on login and registration forms

### ISSUE THREAD

**Issue by gggrant** [GCTC-NTGC/TalentCloud#5160]
Or hidden text that will be picked up by screen readers.

"Show/hide" should be sufficient.
